src/fixturelibrary.ts

```diff
@@ -127,6 +149,25 @@ export class FixtureLibrary {
     return fixture;
   }
 
+  public async getMakeModelIndex(): Promise<MakeModel[]> {
+    const items = await this.fixtureIndex.getIndex();
+    const manufacturers = this.fixtureIndex.getManufacturers();
+    // eslint-disable-next-line no-async-promise-executor
+    return await Promise.all(Object.keys(items).map((key) => new Promise(async (resolve) => {
```

Comment:
Should the output of this be saved as a file? I'm curious how long it takes to run this function vs calculating it once and keeping it in memory like the others.

```suggestion
    return await Promise.all(Object.keys(items).map((key) => async (resolve) => {
```

Also, no need to wrap in a promise.